.gitignore

```diff
@@ -4,3 +4,7 @@
 
 # Conventional directory for build output.
 build/
+
+# Flutter/Dart/Pub related
+# Libraries should not include pubspec.lock, per https://dart.dev/guides/libraries/private-files#pubspeclock.
+pubspec.lock
```

Comment:
Well that is for `dependencies` or packages directly used in the flutter project.

This package is a `CLI` tool, and by official dart template generator it is meant to be version controlled.

Reply:
Resolved in [new commit](https://github.com/obemu/package_rename/commit/23fccf09eeb92780e0bd0e10b45c231503e02128)